addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -995,21 +995,20 @@ def _l10n_it_post_invoices_step_2(self, invoices):
                     if '00404' in error_codes:
                         idx = error_codes.index('00404')
                         invoice.message_post(body=_(
-                            'This invoice number had already been submitted to the SdI, so it is'
-                            ' set as Sent. Please verify that the system is correctly configured,'
-                            ' because the correct flow does not need to send the same invoice'
-                            ' twice for any reason.\n'
-                            ' Original message from the SDI: %s', errors[idx]))
+                            'This invoice number looks duplicate to the SdI.'
+                            ' We set the state as Sent, but please verify the system is correctly configured,'
+                            ' because the correct flow does not allow sending the same invoice twice.\n'
+                            ' Original message from the SdI: %s', errors[idx]))
                         to_return[invoice] = {'attachment': invoice.l10n_it_edi_attachment_id, 'success': True}
                     else:
                         # Add helpful text if duplicated filename error
                         if '00002' in error_codes:
                             idx = error_codes.index('00002')
                             errors[idx] = _(
                                 'The filename is duplicated. Try again (or adjust the FatturaPA Filename sequence).'
-                                ' Original message from the SDI: %s', [errors[idx]]
+                                ' Original message from the SdI: %s', [errors[idx]]
                             )
-                        to_return[invoice] = {'error': self._format_error_message(_('The invoice has been refused by the Exchange System'), errors), 'blocking_level': 'error'}
```

Comment:
Consistency over synonyms. It's not a poem. 
"the SdI" "the SDI" "to SdICoop" "by the Exchange System" "to the Tax Agency"... should all be "SdI".